app/uk/gov/hmrc/customsservicestatus/repositories/CustomsServiceStatusRepository.scala

```diff
@@ -49,7 +49,7 @@ class CustomsServiceStatusRepository @Inject() (
       domainFormat = CustomsServiceStatus.mongoFormat,
       indexes = Seq(
         IndexModel(ascending("id"), IndexOptions().name("serviceIdIdx").unique(true).sparse(true)),
-        IndexModel(ascending("lastUpdated"), IndexOptions().name("lastUpdatedIdx").expireAfter(appConfig.expireAfterSeconds, SECONDS))
+        IndexModel(ascending("lastUpdated"), IndexOptions().name("lastUpdatedIdx"))
```

Comment:
we need to remove the TTL config in the application.conf (and in app-configs if its overridden)

Reply:
updated.